helm/terraform-job/templates/terraform-service-account.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.serviceAccount.name }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
```

Comment:
`helm.sh/hook-weight: "-10"`

Reply:
Made sure all the templates have the hook weight ordering with 49bfe48.